server/views/pages/bookingJourney/selectDateAndTime.njk

```diff
@@ -7,4 +7,6 @@
   BOOKING JOURNEY
   {{ bookingJourney | dump(2) }}
 
+  AVAILABLE VISIT SESSIONS
+  {{ visitSessions | dump(2) }}
 </pre>
```

Comment:
Trailing newline 😛

Reply:
Thought you might pick up on that when I saw the red marker 😂 ! Don't worry: it'll be gone soon...